### PR TITLE
Update microsoft-365-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed link for M365 business premium as it was routing to admin center help

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9286

File: microsoft-365/enterprise/microsoft-365-overview.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ For an example of how a fictional but representative multinational organization
 
 ## Additional Microsoft 365 products
 
-- [Microsoft 365 Business Premium](../admin/index.yml)
+- [Microsoft 365 Business Premium](https://docs.microsoft.com//microsoft-365/business/)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-overview.md]:
```suggestion
- [Microsoft 365 Business Premium](/microsoft-365/business/)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**